frontend/src/component/feature/FeatureView/FeatureOverview/FeatureOverviewEnvironments/FeatureOverviewEnvironment/EnvironmentAccordionBody/EnvironmentAccordionBody.tsx

```diff
@@ -36,11 +36,25 @@ const StyledAccordionBodyInnerContainer = styled('div')(({ theme }) => ({
     },
 }));
 
-export const StyledContentList = styled('ol')({
+export const StyledContentList = styled('ol')(({ theme }) => ({
     listStyle: 'none',
     padding: 0,
     margin: 0,
-});
+
+    '& > li': {
+        paddingBlock: theme.spacing(2.5),
+        position: 'relative',
+    },
+    '&:not(li > &) > li:first-of-type': {
+        // select first list elements in lists that are not directly nested
+        // within other lists.
+        paddingTop: theme.spacing(1),
+    },
+    '& > li:has(> ol)': {
+        // nested lists add their own padding to their list items, so we don't want to double it up.
+        paddingBlock: 0,
+    },
```

Comment:
Yay, CSS selectors 🥳 I moved all the li spacing and positioning up here so that it's colocated.